docs/queries/filters.md

```diff
@@ -440,6 +442,9 @@ Introduced in Tasks 1.6.0.
   - The trailing `i` means case-insensitive.
   - Searches for tags such as `#book`,  `#Book`, `#BOOK` and the `$` prevents matching of `#books`,  `#book/literature`, etc.
 
+{: .released }
+`has tags` and `no tags` were introduced in Tasks 1.26.0.
```

Comment:
I think this should be in the previous `### Tags` section.

Reply:
> Thanks! Just a couple of tweaks to the docs, please.

Oh great, I'm being so focused on the code that my attention goes elsewhere when writing the docs. Thanks so much.

Pls resolve the conversations if looks fine for you =)